.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an enhancement to the ohsome API.
+title: ''
+labels: 'feature request'
+assignees: ''
+
+---
+
+### Use Case Description
+Describe the use case which would benefit from this enhancement.
+
+### Request Description
+Add a clear description about the enhancement. Please be specific about the features you would like to have.
+
+### Additional Information
+Add any other information, examples, screenshots, etc. about the feature request here.
```

Comment:
to address https://github.com/GIScience/ohsome-api/pull/159#discussion_r608502509, let's perhaps write it like this:

```suggestion
### Request Description
Add a clear description about the enhancement. Please be specific about the feature you would like to have.

### Additional Information
Add any other information, examples, screenshots, possible workarounds, etc. about the feature request here. You may indicate here how important the enhacement is for your use case and whether or not you would be willing to support the development (from coding a pull request, to helping as a beta tester).
```

?

Reply:
That is fine, but I think the last part "and whether or not..." it doesn't add any important information since we code by ourselves. If we need for some help (e.g. a beta tester) we can just ask him/her.

Reply:
Well, it's an open source project after all. So it could be the case that at some point someone external might want to contribute. :)
Or, for example, if let's say that @SlowMo24 really needs a certain feature very urgently ( :wink: ), it could speed up some thinks if we know that we can count on active support.

Or if a feature request comes from a team member, but who has not time/skill/… to implement a certain feature, it would also be good to know, IMO.

Reply:
I see the point, just I don't think the text adds really value. But I have no problem with adding this text.;-)